### PR TITLE
MI2_Assignment01

### PR DESCRIPTION
**Pull request by SudeshnaBora**

### CODE REVIEW

File: mi2/Assignment01/pca-data-2d.dat.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+    2.4000   -2.5000
+    0.7000   -0.5000
+    2.9000   -2.2000
+    2.2000   -1.9000
```
Comment (Konstantine00) [inline mi2/Assignment01/pca-data-2d.dat.txt:4]: Why is this file name .dat.txt? also is it possible to rename the assignment folder to lower case (semantic issues)

File: mi2/Assignment01/imgpca/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Konstantine00) [inline mi2/Assignment01/imgpca/README.md:1]: is this file necessary?

File: mi2/Assignment01/pca-data-2d.dat.txt (hunk 1 above)
Comment (SudeshnaBora) [reply mi2/Assignment01/pca-data-2d.dat.txt:4]: Cause I could not download the .dat file . So I copied the contents and made my own file.

File: mi2/Assignment01/imgpca/README.md (hunk 2 above)
Comment (SudeshnaBora) [reply mi2/Assignment01/imgpca/README.md:1]: no
